ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -339,4 +342,26 @@ Find the transaction on https://explorer.ironfish.network/transaction/${transact
       this.exit(2)
     }
   }
+
+  async doElegibilityCheck(): Promise<void> {
+    const api = new WebApi()
+    const graffiti = this.sdk.config.get('blockGraffiti')
+    const user = await api.findUser({ graffiti: graffiti })
+    if (!user) {
+      this.log(`WARNING: Could not find a user with graffiti ${graffiti}`)
+    } else {
+      if (!user.is_verified) {
+        this.log(
+          `WARNING: No verified email on account for graffiti ${graffiti}. You need this email to claim testnet rewards. Visit https://ironfish.network and click "Log In" to verify.`,
+        )
+      }
+      if (user.node_uptime_score < 14) {
```

Comment:
Can we just encode this here so we can change this in the API if we need to?

https://github.com/iron-fish/ironfish-api/pull/1227/files

Reply:
What if we send back 

total_hours and total_hours_needed, or what ever you want to name it just under that principal. Once this goes out we can't really change it easily since people aggressively use older versions.

Reply:
done.